doc/user_manual/optimizer.tex

```diff
@@ -66,8 +66,9 @@ \subsection{Gradient Based Optimizers}
 different sampling methodologies are reported.
 
 Note that in addition to the input variables and response variable, several other parameters are available to
-request for the output of a Gradient-Based Optimizer run.  Note that each starts with an underscore.  They include the following:
+request for the output of a Gradient-Based Optimizer run.  Note that many start with an underscore.  They include the following:
```

Comment:
I agree, my original justification for the naming of these variables was not great.  I'll remove the underscores and use camelBack, except in the case of "gradient_<var>", where the underscore serves to protect the user variable in clarity.